app/src/main/java/org/kiwix/kiwixmobile/zim_manager/fileselect_view/effects/DeleteFiles.kt

```diff
@@ -40,17 +40,25 @@ data class DeleteFiles(private val booksOnDiskListItem: List<BookOnDisk>) :
 
   override fun invokeWith(activity: AppCompatActivity) {
     (activity as ZimManageActivity).cachedComponent.inject(this)
-    booksOnDiskListItem.forEach {
-      dialogShower.show(DeleteZim(it), {
-        if (deleteSpecificZimFile(it)) {
-          if (it.file.canonicalPath == zimReaderContainer.zimCanonicalPath) {
-            zimReaderContainer.setZimFile(null)
-          }
-          activity.toast(R.string.delete_specific_zim_toast)
-        } else {
-          activity.toast(R.string.delete_zim_failed)
+
+    val name = booksOnDiskListItems.joinToString(separator = "\n") { it.book.title }
+
+    dialogShower.show(DeleteZims(name), {
+      activity.toast(
+        if (booksOnDiskListItems.deleteAll())
```

Comment:
formatting should be a bit different here

Reply:
@macgills The commit looks didn't allow for it to be in a single line. So I had to shift it to multiple lines.

Reply:
The autoformatter is not infallible and doesn't have an eye for beauty it must be said.

my strategies with `if`s as an expression is to not use braces
try to fit on one line
```
if (condition) value1 else value2
```
if that is too long
```
if(condition) value1
else value2
```
and if that is too long 

```
if(condition)
  value1
else
  value2
```

but the world is not going to end because this if has braces so I'll go ahead with the merge